### PR TITLE
Cleaned up TCL code, added more documentation to configs

### PR DESCRIPTION
**Pull request by JesseSandberg**
I added some more documentation to the config files, fixed some small errors, and generally linted.

There are really only two substantive changes:

1) the user now needs to specify the location of the repo on their respective computer. This allows them to run the calculations from wherever they want, not just in the repo itself.
2) we were trying to call a global variable ($USE_QWRAP) that wasn't actually global. I made it global.

### CODE REVIEW

**Review by EzryStIago — CHANGES_REQUESTED** [review]
That's not what `variable` does. The utility of this change isn't clear to me.
Please clarify what issue this fixes or revert the change.

File: TCL/polarDensity_for_DTA.tcl [hunk 1]
```diff
@@ -320,7 +302,6 @@ proc local_mid_plane2 {atsel_in frame_i} {
     }
 }
 
-
 proc leaflet_sorter_0 {atsel_in head tail frame_i} {
```
Comment (EzryStIago) [inline TCL/polarDensity_for_DTA.tcl:305]: The atomselections are deleted after we return (the `$sel delete` lines never get executed)

**Review by EzryStIago — CHANGES_REQUESTED** [review]
Can you fix issue #20 and clarify your change from `set` to `variable`?

**Comment by JesseSandberg** [comment]
> That's not what `variable` does. The utility of this change isn't clear to me. Please clarify what issue this fixes or revert the change.

I was getting an error message because center_system doesn't know what USE_QWRAP is. Changing set `set` to `variable` fixed it.

File: TCL/polarDensity_for_DTA.tcl (hunk 1 above)
Comment (JesseSandberg) [reply TCL/polarDensity_for_DTA.tcl:267]: Should be fixed now

**Review by EzryStIago — APPROVED** [review]

**Pull request merged**